oak_runtime/src/node/http/tests.rs

```diff
@@ -159,34 +145,35 @@ fn create_runtime() -> RuntimeProxy {
         wasm_modules: hashmap! {},
         initial_node_configuration: None,
     };
-    let signature_table = crate::SignatureTable::default();
-    info!("Create runtime for test");
-
-    crate::RuntimeProxy::create_runtime(
-        &configuration,
-        &crate::SecureServerConfiguration::default(),
-        &signature_table,
-    )
-}
-
-fn create_server_node(port: u32) -> Box<HttpServerNode> {
     let tls_config = crate::tls::TlsConfig::new(
         "../examples/certs/local/local.pem",
         "../examples/certs/local/local.key",
     )
     .expect("Could not create TLS config from local certs.");
+    let secure_server_config = crate::SecureServerConfiguration {
+        grpc_config: None,
+        http_config: Some(crate::HttpConfiguration { tls_config }),
+    };
+    let signature_table = crate::SignatureTable::default();
+    info!("Create runtime for test");
 
-    // Create http server node
-    Box::new(
-        HttpServerNode::new(
-            "test-node",
-            HttpServerConfiguration {
-                address: format!("[::]:{}", port),
-            },
-            tls_config,
-        )
-        .expect("Could not create server node"),
-    )
+    crate::RuntimeProxy::create_runtime(&configuration, &secure_server_config, &signature_table)
+}
+
+fn create_server_node(runtime: &RuntimeProxy, port: u32) -> oak_abi::Handle {
+    let (init_receiver, invocation_receiver) = create_communication_channel(runtime);
```

Comment:
Could this (or the underlying method) use the typed version of `channel_create` with actual types? Since below you are wrapping that anyways into a typed Receiver. BTW we should probably clarify that that is the preferred one, instead of the one that returns "raw" handles.

Reply:
The one in the `sdk`? Can I use that one or is there another one in the Runtime?

Reply:
Good point, I think it's missing from the Runtime, could you create an issue for it so we don't forget? Perhaps it could go in `oak_runtime::io`.

Reply:
Created #1496. I won't add a TODO here, because there are many other places in the code that should be updated as well.